src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
The `Main` class should not begin with an empty line after the class declaration according to the checklist.